benchmark/benchmarks.jl

```diff
@@ -1,18 +1,36 @@
-using BenchmarkTools
-using Graphs
+using BenchmarkTools, Graphs
 
-DIGRAPHS = Dict{String,DiGraph}(
+const BENCHDIR = dirname(@__FILE__)
+
+const DIGRAPHS = Dict{String,DiGraph}(
     "complete100" => complete_digraph(100), "path500" => path_digraph(500)
 )
 
-GRAPHS = Dict{String,Graph}(
+const GRAPHS = Dict{String,Graph}(
     "complete100" => complete_graph(100),
     "tutte" => smallgraph(:tutte),
     "path500" => path_graph(500),
 )
 
-suite = BenchmarkGroup()
-include("core.jl")
+serialbenchmarks = [
+    "serial/core.jl",
+    "serial/connectivity.jl",
+    "serial/centrality.jl",
+    "serial/edges.jl",
+    "serial/insertions.jl",
+    "serial/traversals.jl",
+]
+
+const SUITE = BenchmarkGroup()
+
+foreach(serialbenchmarks) do bm
+    include(bm)
+end
+
+parallelbenchmarks = ["parallel/egonets.jl"]
+
+foreach(parallelbenchmarks) do bm
+    include(joinpath(BENCHDIR, bm))
+end
 
-tune!(suite);
-results = run(suite; verbose=true, seconds=10)
+nothing
```

Comment:
?

Reply:
otherwise the `results` variable is printed when running tests (e.g. with `include("test/runtests.jl")`).

Reply:
At this case for `include("benchmark/benchmarks.jl")`

Reply:
You can also run include with a semicolon at the end if you want, so I would remove the nothing

Reply:
okey.